framework/db/ActiveRecord.php

```diff
@@ -268,7 +268,7 @@ public static function deleteAll($condition = '', $params = [])
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQuery::className(), [get_called_class()]);
+        return Yii::createObject(ActiveQuery::class, [get_called_class()]);
```

Comment:
I mean `ActiveQuery`. In this case `find()` is called on AR instance and returns AQ instance. The difference is how it's created.